src/classes/application.js

```diff
@@ -160,6 +161,14 @@ module.exports = class Application extends Base {
     const app = express()
     _app.set(this, app)
 
+    log.trace('Ash server loading cors middleware')
+    if (typeof config.cors === 'object') {
+      if (config.cors.preFlight === true) {
```

Comment:
The options handling for cors where you use options to ask what cors stuff is supported.